db/src/db.rs

```diff
@@ -390,6 +393,24 @@ pub fn read_db(conn: &rusqlite::Connection) -> Result<DB> {
     Ok(DB::new(partition_map, schema))
 }
 
+use itertools;
```

Comment:
Also, re: file length -- yes, this is long.  I have Grand Plans for a split between the DB layer and the transaction processing layer but I don't want to make the split until more of the transactor is in place.  So eventually I expect two files of similar size, but right now it's just one big file.  Sorry!